Framework/Core/Windows/Devices/Pwm/PwmController.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using Windows.Devices.Pwm.Provider;
+
+namespace Windows.Devices.Pwm
+{
+    public sealed class PwmController
+    {
+        private IPwmControllerProvider m_provider;
+        private double m_minFrequency;
+        private double m_maxFrequency;

```

Comment:
As you said, the value of these two properties are cached while the value of other properties are not, e.g. pin count. So basically there is a difference between the behavior of these two properties and the others. If this difference is expected keep them as they are; otherwise, change them to match the others (or vice versa).